tsl/test/expected/decompress_vector_qual.out

```diff
@@ -165,8 +165,10 @@ select * from vectorqual where ts > '2021-01-01 00:00:00' and metric3 > 40 order
  Wed Jan 01 00:00:00 2025 |      52 |      5 |      53 |      54 | tag5
 (2 rows)
 
--- ORed constrainst on multiple columns (not vectorized for now).
-set timescaledb.debug_require_vector_qual to 'forbid';
+-- ORed constrainst on multiple columns.
+set timescaledb.debug_require_vector_qual to 'only';
+-- set timescaledb.debug_require_vector_qual to 'forbid';
```

Comment:
Lines 170 + 171 look like debug lines. Are they removed in the final PR?

Reply:
This is to get the test output w/o vectorized quals. I tend to uncomment it often when I'm adding tests, so I'd just leave it there.